src/app.css

```diff
@@ -23,17 +20,14 @@
     --secondary-element-color-lt: rgba(0, 0, 0, 0.1);
     --accent-text-color-lt: #0080c8;
```

Comment:
as far as I know the Tailwind has themes switching configuration under the hood. The same rule for modals and so. Don't consider it as "just a feature what gives me ability to write css in jsx" it's wrong. And I really think it digression, but who asks me :)